src/Adapter/CMS/PageCategory/CommandHandler/BulkDisableCmsPageCategoryHandler.php

```
@@ -0,0 +1,70 @@
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Adapter\CMS\PageCategory\CommandHandler;
+
+use CMSCategory;
+use PrestaShop\PrestaShop\Core\Domain\CmsPageCategory\Command\BulkDisableCmsPageCategoryCommand;
+use PrestaShop\PrestaShop\Core\Domain\CmsPageCategory\CommandHandler\BulkDisableCmsPageCategoryHandlerInterface;
+use PrestaShop\PrestaShop\Core\Domain\CmsPageCategory\Exception\CannotBulkDisableCmsPageCategoryException;
+use PrestaShop\PrestaShop\Core\Domain\CmsPageCategory\Exception\CmsPageCategoryException;
+use PrestaShopException;
+
+/**
+ * Class BulkDisableCmsPageCategoryHandler is responsible for deleting multiple cms page categories.
+ */
+class BulkDisableCmsPageCategoryHandler implements BulkDisableCmsPageCategoryHandlerInterface
+{
+    /**
+     * {@inheritdoc}
+     *
+     * @throws CmsPageCategoryException
+     */
+    public function handle(BulkDisableCmsPageCategoryCommand $command)
+    {
+        try {
+            foreach ($command->getCmsPageCategoryIds() as $cmsPageCategoryId) {
+                $entity = new CMSCategory($cmsPageCategoryId->getValue());
+                $entity->active = false;
```

you should throw exception if `CMSCategory` is not found before trying to update it.